tutorials/katacoda/thanos/2-lts/step1.md

```diff
@@ -0,0 +1,150 @@
+# Step 1 - Initial Prometheus Setup
+
+Thanos is a set of components that adds high availability to Prometheus installations by the cross-cluster federation, unlimited storage, and global querying across clusters.
+
+Thanos builds upon vanilla Prometheus instances which states it seamlessly integrates into existing Prometheus setups.
+
+In this tutorial, we will be using one Prometheus Server in the eu1 region.
+
+Let's start this initial Prometheus setup, ready?
+
+
+## Generate Artifical Metric Data
+
+Execute the following command:
+
+```
+mkdir test
+```{{execute}}
+
+```
+docker run -i dockerenginesonia/thanosbench:v1.4 block plan -p realistic-key-k8s-1d-small --labels 'cluster="one"' --max-time 2019-10-18T00:00:00Z | docker run -i dockerenginesonia/thanosbench:v1.4 block gen --output.dir test
```

Comment:
you need to add a docker volume to actually write it to the test directory, like in here https://github.com/thanos-io/thanos/pull/2840/files#diff-3ebc19a6a112ac8bb9c23b456789d37fR71